test/deploy/os_ova_install.py

```diff
@@ -49,15 +64,18 @@ def test01_install_ova_template(self):
                                 timeout=20, logfile=None)
             if exitstatus == 0:
                 break
-
         # Run OVA installer
+        for vm in range(0, numvms):
+            self.deploy_ova(vm,uname,passwd,numvms,ovafile)
+
+    def deploy_ova(self,vm,uname,passwd,numvms,ovafile):
```

Comment:
I don't think this should go inside the test.  That makes it harder to read.  It should be inside the class but outside of test01_install_ova_template.  Perhaps before the definition of test01_install_ova_template.